src/main/java/cloud/fogbow/as/core/models/CloudStackSystemUser.java

```diff
@@ -2,8 +2,17 @@
 
 import cloud.fogbow.common.models.CloudStackUser;
 
+import java.util.HashMap;
+
 public class CloudStackSystemUser extends OneToOneMappableSystemUser {
+    private HashMap<String, String> cookieHeader;
```

Comment:
This field could be named 'cookieHeaders' as well.

Reply:
ok